src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/ComplexityOfNeedAPI.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.keyworker.services
+
+import org.springframework.core.ParameterizedTypeReference
+import org.springframework.http.MediaType
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.keyworker.events.ComplexityOfNeedLevel
+import java.time.LocalDateTime
+
+data class ComplexOffender(
+  val offenderNo: String,
+  val level: ComplexityOfNeedLevel,
+  val sourceUser: String? = null,
+  val sourceSystem: String? = null,
+  val notes: String? = null,
+  val createdTimeStamp: LocalDateTime? = null
+)
+
+@Service
+class ComplexityOfNeedAPI(private val complexityOfNeedWebClient: WebClient) {
```

Comment:
Don't know if the "*API" suffix is standard naming, but at previous places I have seen these server classes called "*Gateway" or "*Client"